docs/content/drafts/ActionList2.mdx

```diff
@@ -43,163 +43,139 @@ import InlineCode from '@primer/gatsby-theme-doctocat/src/components/inline-code
 </Box>
 
 ```js
-import {ActionList} from '@primer/react/drafts'
+// ESM
+import {ActionList} from '@primer/react/lib-esm/drafts'
+// or commonjs:
+import {ActionList} from '@primer/react/lib/drafts'
```

Comment:
Hey I think after this https://github.com/primer/react/pull/1771,
we don't need the `lib-esm/drafts`anymore

Reply:
How, that's brilliant! I can change all the imports once #1771 is merged 🎉

Reply:
Talked to @pksjce about https://github.com/primer/react/pull/1771. There will be be some more improvements after that PR is merged 🎉  Until then, this is a step in the right direction, so merged :)